<launch>
  <include file="$(find openni_launch)/launch/openni.launch">
    <arg name="camera" value="kinect1" />
  </include>
  
  <node pkg="kinfu" type="kinfu" name="kinect_kinfu1" output="screen">
    <param name="prefix_topic" value="/kinect1" />
    <param name="autostart" value="true" />
    <param name="response_topic" value="/kinfu1_response_topic" />
    <param name="request_topic" value="/kinfu1_request_topic" />
    <param name="extract_known_points" value="true" />
  </node>

  <node pkg="kinfu_output" type="kinfu_output" name="kinect_kinfu1_output" output="screen">
    <param name="kinfu_response_topic" value="/kinfu1_response_topic" />
  </node>
</launch>
